pages/api/legacy/contribute.js

```diff
@@ -0,0 +1,9 @@
+import path from 'path';
+
+export default function handler(req, res) {
+  const color = req.query.color === 'blue' ? 'blue' : 'white';
+  const verb = ['donate', 'contribute'].includes(req.query.verb) ? req.query.verb : 'contribute';
+  const size = req.query.size === '2x' ? req.query.size : '';
```

Comment:
Given that the size is supposed to include a `@` in the URL, are'nt we supposed to use it here as well? 

https://github.com/opencollective/opencollective-frontend/blob/main/public/static/images/buttons/contribute-button-blue%402x.png

```suggestion
  const size = req.query.size === '@2x' ? req.query.size : '';
```

Reply:
Good catch